rs/pocket_ic_server/BUILD.bazel

```diff
@@ -247,7 +247,7 @@ rust_test(
     # this test is manual to not block automatic PR merge
     # bumping mainnet canister versions
     # from which the external types are derived
-    tags = ["manual"],
+    tags = ["pocketic_tests_nightly"],
```

Comment:
Why is this test also [running](https://dash.zh1-idx1.dfinity.network/invocation/8f2921b9-aeef-4aa1-b372-eab29740afca?target=%2F%2Frs%2Fpocket_ic_server%3Aexternal_types_tests&targetStatus=6) on PRs?